com.adilhanney.ricochlime.json

```diff
@@ -0,0 +1,85 @@
+{
+	"app-id": "com.adilhanney.ricochlime",
+	"runtime": "org.freedesktop.Platform",
+	"runtime-version": "22.08",
+	"sdk": "org.freedesktop.Sdk",
+	"command": "ricochlime",
+	"separate-locales": false,
+	"finish-args": [
+		"--share=network",
```

Comment:
does it need network?

Reply:
For now, it does for the [google_fonts](https://pub.dev/packages/google_fonts) package. In the future, I'll include the fonts offline

Reply:
they should be in the package instead.